cpp/Platform.Data.Doublets/ILinksExtensions.h

```diff
@@ -291,13 +291,13 @@ namespace Platform::Data::Doublets
                 }
             }
         }
-        
+
```

Comment:
Please make your commits atomic. For example: when you format code - do that in a different commit, not in a mess with other changes